addons/l10n_id/data/ir_cron.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="qris_fetch_cron" model="ir.cron">
+        <field name="name">QRIS Fetch Status</field>
+        <field name="model_id" ref="account.model_account_move"/>
+        <field name="interval_number">1</field>
+        <field name="interval_type">hours</field>
```

Comment:
We call it every hour, but there is a timeout of 30min on QRIS?

Reply:
Hi! The timeout of 30 minutes is only for the QR code payment (after generating the QR code, there is a 30m window to pay. After that, the payment would be refused).
According to my tests, the payment statuses are available at least 24h.